tests/baselines/reference/JSDocParsing/DocComments.parsesCorrectly.typedefTagWithChildrenTags.json

```diff
@@ -20,30 +20,14 @@
                 "transformFlags": 0,
                 "escapedText": "typedef"
             },
-            "fullName": {
-                "kind": "Identifier",
-                "pos": 17,
-                "end": 23,
-                "modifierFlagsCache": 0,
-                "transformFlags": 0,
-                "escapedText": "People"
-            },
-            "name": {
-                "kind": "Identifier",
-                "pos": 17,
-                "end": 23,
-                "modifierFlagsCache": 0,
-                "transformFlags": 0,
-                "escapedText": "People"
-            },
             "typeExpression": {
                 "kind": "JSDocTypeLiteral",
                 "pos": 8,
                 "end": 100,
                 "modifierFlagsCache": 0,
                 "transformFlags": 0,
-                "jsDocPropertyTags": [
-                    {
+                "jsDocPropertyTags": {
```

Comment:
??? Why'd this go from being serialized as an array to an object with numeric keys? Are `NodeArray`s no longer arrays?

Reply:
That's how JSON serialization handles arrays that have extra properties, so the properties aren't lost.